jen/statement.go

```diff
@@ -72,8 +72,22 @@ func (s *Statement) render(f *File, w io.Writer, _ *Statement) error {
 // Render renders the Statement to the provided writer.
 func (s *Statement) Render(writer io.Writer) error {
 	f := NewFile("")
+	return s.RenderWithFile(writer, f)
+}
+
+// GoString renders the Statement for testing. Any error will cause a panic.
+func (s *Statement) GoString() string {
+	buf := bytes.Buffer{}
+	if err := s.Render(&buf); err != nil {
+		panic(err)
+	}
+	return buf.String()
+}
+
+// RenderWithFile renders the Statement to the provided writer
```

Comment:
Also, let's change the comment to "RenderWithFile renders the Statement to the provided writer, using imports from the provided file."

Reply:
Fixed, thanks